xsd/netex_part_1/part1_networkDescription/netex_routeInstruction_version.xsd

```diff
@@ -73,7 +73,12 @@ Rail transport, Roads and Road transport
 		<xsd:complexContent>
 			<xsd:extension base="strictContainmentAggregationStructure">
 				<xsd:sequence>
-					<xsd:element ref="RouteInstruction" maxOccurs="unbounded">
+					<xsd:element ref="RouteInstructionRef" minOccurs="0" maxOccurs="unbounded">
```

Comment:
This is still not what we want. Sorry.

Is a choice with minOccurs=1 (I don't know if this is possible)
And the sub elements of choice should also have a minOccurs=1. I'll open a separate issue.

Reply:
Remove the sequence.
```
<choice maxOccurs="unbounded">
  <xsd:element ref="RouteInstructionRef">
  ...
  <xsd:element ref="RouteInstruction">
</choice>
```

I do want to have the discussion if we can actually generically do something else being:

```
<choice maxOccurs="1">
  <sequence>
    <xsd:element ref="RouteInstructionRef" maxOccurs="unbounded">
  </sequece>
  ...
  <sequence>
    <xsd:element ref="RouteInstruction"  maxOccurs="unbounded">
  </sequence>
</choice>
```

Reply:
second solution to be implemented.

Reply:
done